pkg/tanka/parallel.go

```diff
@@ -9,10 +9,10 @@ import (
 
 const parallel = 8
 
-// ParseParallel evaluates multiple environments in parallel
-func ParseParallel(paths []string, opts ParseParallelOpts) (envs []*v1alpha1.Environment, err error) {
+// parallelLoadEnvironments evaluates multiple environments in parallel
+func parallelLoadEnvironments(paths []string, opts ParallelOpts) (envs []*v1alpha1.Environment, err error) {
```

Comment:
```suggestion
func parallelLoadEnvironments(paths []string, opts ParallelOpts) ([]*v1alpha1.Environment, error) {
```

Named returns should generally not be used, unless they serve specific documentational purpose. They don't seem to do that here, as it's clear what this function returns

Reply:
Only review cosmetic changes in this PR please.